conmon-rs/server/src/version.rs

```diff
@@ -40,7 +39,7 @@ impl Version {
 
     /// Print the version information to stdout.
     pub fn print(&self) {
-        println!("{} version {}", crate_name!(), build::PKG_VERSION);
+        println!("version: {}", build::PKG_VERSION);
```

Comment:
That was the non-breaking change to allow version parsing in c/common. I think we can skip that now.